packages/core-app-api/report.api.md

```diff
@@ -478,6 +491,15 @@ export class LocalStorageFeatureFlags implements FeatureFlagsApi {
   save(options: FeatureFlagsSaveOptions): void;
 }
 
+// @public
+export type LoginPopupOptions = {
```

Comment:
Rename this one to `ShowLoginPopupOptions`, or matching w/e name the function ends up with